### PR TITLE
Add localization support

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

Add localization support for `mgradm`, `mgrctl` and `mgrpxy` with some minimal french translation to make sure it works.

## Test coverage
- No tests: localization tests would be too flaky

- [X] **DONE**

## Links

Issue(s): https://github.com/uyuni-project/uyuni-tools/issues/132

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Review by mbussolotto — APPROVED** [review]
I would put `check_localizable` also in `build.sh` , to simplify the check locally. The rest is fine

File: uyuni-tools.spec [hunk 1]
```diff
@@ -40,7 +40,7 @@
 %define _zshdir %{_datarootdir}/zsh/vendor-completions
 %else
 %define _zshdir %{_datarootdir}/zsh/site-functions
-%endif
+%endif # %if 0%{?debian} || 0%{?ubuntu}
```
[the remark this answers is not in the document]
Comment (cbosdo) [reply uyuni-tools.spec:43]: hum... you're killing my dream

[the remark this answers is not in the document]
Comment (cbosdo) [reply uyuni-tools.spec:43]: fixed

Comment (deneb-alpha) [inline uyuni-tools.spec:43]:
```suggestion
%endif # 0%{?debian} || 0%{?ubuntu}
```

File: uyuni-tools.spec [hunk 2]
```diff
@@ -234,12 +237,23 @@ Requires:       %{name_ctl} = %{version}
...
+%endif # 0%{?suse_version} >= 150000
 BuildArch:      noarch
 
 %description -n %{name_ctl}-fish-completion
 Fish command line completion support for %{name_ctl}.
-%endif
+%endif # 0%{?is_opensuse} || 0%{?fedora} || 0%{?debian} || 0%{?ubuntu}
+
+# Only SUSE distros have a -lang packages, for the others they
+# will all be in the correspdonding tool package.
```
Comment (deneb-alpha) [inline uyuni-tools.spec:248]: dumb question, but... why?

**Review by deneb-alpha — COMMENTED** [review]
a few comment/suggestion

File: uyuni-tools.spec (hunk 2 above)
Comment (cbosdo) [reply uyuni-tools.spec:248]: why SUSE distros have `-lang` packages or why the others don't? SUSE provides RPM macros to define the lang packages with a single line each while those don't exist in the other distros and would require a full description for each.

**Review by deneb-alpha — APPROVED** [review]
let's give it a try! :) ready to localize!

**Pull request merged**